.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,3 +7,8 @@
 
 <!-- How was this change tested? -->
 <!-- DON'T DELETE THIS SECTION! If no tests were added, explain why. -->
+
+## Docs
+
+<!-- Have new features been added or existing behavior changed? Please state places the changes were documented -->
```

Comment:
100% on board with this idea, I forget to add docs all the time.

The longer these lines are, the more likely someone is to not read. Can you shrink to match the lines above? "Please state places the changes were documented" feels unnecessary since it should be apparent in the PR. 

How about:

```
<!-- Was "docs/" documentation site updated? -->
```

Reply:
I'm not sure if that will suffice. Not all documentation would (only) go into the `docs/` section. For example, if someone implements a new feature for a snowpack plugin, the documentation would go into its README.md file.

I agree shorter is better, but I would like to convey that there can be multiple places where docs should be / have been updated

Reply:
Yup, feel free to take a crack at it yourself

Reply:
I'll shorten it to 

```suggestion
<!-- List places where documentation was updated, if applicable -->
```

Reply:
Sorry I could have been more specific, when I meant match I did mean match the question format of the others as well. Nit, I know, but consistency does show care. I'll push a quick change to match the form, and then take a look and let me know if you have any feedback on it.